### DISCUSSION TITLE
Is it possible to connect a SIP *phone* to ejabberd?

### DISCUSSION THREAD

**Discussion by alexpdp7** [processone/ejabberd#3589] [Questions] [answered]
I have an ejabberd server setup, and I'd like to use hardware SIP phones with it. I see there's `mod_sip` et al. but it's not clear if I can do things such as call an XMPP client (like Conversations) from a hardware SIP phone, or interop with the general SIP world (e.g. connect to the PSTN?).

**Comment by mremond** [comment, marked as answer]
The SIP module in ejabberd allows you to connect SIP phone and pass call between SIP phones connected on ejabberd. It will share the same user base / credentials.

At the moment, there is no SIP-XMPP bridge and no support for using SIP gateways for PSTN call.
